webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -1775,6 +1779,63 @@ var MCK_MAINTAIN_ACTIVE_CONVERSATION_STATE;
                     }
                 });
             };
+
+            _this.configureRatingElements = function(){
+                var ratingSmilies = document.getElementsByClassName("mck-rating-box");
+                var sendFeedbackComment = document.getElementById('mck-submit-comment');
+                var restartConversation = document.getElementById('mck-restart-conversation');
+                var feedbackObject = {
+                    groupId:0,
+                    comments:[],
+                    rating:0
+                };
+
+                restartConversation.addEventListener('click',function(){
+                    KommunicateUI.showClosedConversationBanner(false);
+                })
+
+                sendFeedbackComment.addEventListener('click',function(){
+                    feedbackObject.comments = [document.getElementById("mck-feedback-comment").value];
+                    feedbackObject.rating = CURRENT_GROUP_DATA.currentGroupFeedback.rating;
+                    feedbackObject.groupId = CURRENT_GROUP_DATA.tabId;
+                    mckUtils.ajax({
```

Comment:
@akshatdhabi van this `ajax` call and the one on the line `1821` be moved to a common function since both the code is almost identical?